articles/azure-maps/toc.yml

```diff
@@ -315,84 +305,6 @@ items:
       href: supported-browsers.md
     - name: Best practices
       href: web-sdk-best-practices.md
-  - name: Develop with the Android SDK
-    items:
-    - name: Android SDK migration guide
```

Comment:
Besides remove this migration guide from TOC, do we consider to reference this article link in other articles under Develop with Web SDK? I'm thinking of different options:
1. Just remove from TOC and not link to this article.
2. Remove from TOC, have a Notes or Remark for providing link to this article for reader who had interests to know Android and iOS SDK migration.
Could you provide some suggestion on this? @stevemunk

Reply:
#1 unless you can think of a reason it needs to remain available?